services/inventory/agents_test.go

```diff
@@ -73,6 +74,7 @@ func TestAgents(t *testing.T) {
 
 		as.r.(*mockRegistry).On("IsConnected", "/agent_id/00000000-0000-4000-8000-000000000001").Return(true)
 		as.r.(*mockRegistry).On("SendSetStateRequest", ctx, "/agent_id/00000000-0000-4000-8000-000000000001")
+		as.r.(*mockRegistry).On("CheckConnectionToService", ctx, mock.Anything, mock.Anything).Return(nil)
```

Comment:
Can we avoid `Anything`?

Reply:
unfortunately, we can change it with only AnythingTypeOf(). 
Params required here are *models.Service and *models.Agent. Both of them have CreatedAt and UpdatedAt fields with current time value. 
Any idea how to handle it?